fallout/smoke-test-fallout.yaml

```diff
@@ -0,0 +1,133 @@
+# region setup
+astra_provider: aws
+astra_region: eu-west-1
+
+# client setup
+client_count: 1
+client_provider: ec2
+client_region: eu-west-1
+client_type: m5.2xlarge
+client_nb_version: 5.17.3-release
+
+# keyspace setup
+keyspace: jsonapi_smoke_test
+
+# documents 10K to enable sorting
+docs_count: 10000
+
+# load setup
+op_cycles: 100000
+op_rate: 25
+op_connections: 3
+
+# ops from main block of smoke-test-nosqlbench.yaml
+ops:
+  - op: find-one-id
+  - op: find-one-by-city
+  - op: find-multi-by-married-name-exists
+  - op: find-multi-by-married-name-exists-with-projection
+  - op: find-multi-find-by-gender-sort-by-name
+  - op: find-by-name-newest-and-update-city
+  - op: delete-one-id
+  - op: find-one-id-and-replace-upsert
+  - op: count-all
+  - op: count-by-gender
```

Comment:
@vkarpov15 We agreed to ask you about a more realistic scenario structure.. Currently we run these in parallel, and each weights the same, so each op contributes with 10%.. Also it would be great if you have any additions to this, some operation we should include, etc?

Reply:
@ivansenic it might be helpful to add insert operations, and maybe some more update operations. In particular, an updateMany. However, Mongoose apps currently tend to be read heavy, so this isn't too far off.

I think we should take a look at the results of this scenario, because this scenario covers a good range of features. If there are performance issues, it's worth investigating.

In the future, we can add scenarios that are more in line with basic apps, like simulating what happens with our sample apps under load.

Reply:
I think @maheshrajamani found one thing that may help improve performance (count(1) instead of count(column)), we'll see how much that helps (fix has been merged).

Reply:
* Added update many instead of the count all
* Adapted count and sort ops so that we can control how many docs should be targeted in those ops